### PR TITLE
ARRAY_VIEW & MATRIX_VIEW node

### PR DESCRIPTION
**Pull request by Jin1L**
### Description

- [x] I've included a concise description of what each node does

### Styleguide

- [x] My node adheres to the [styleguide](https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md) for Flojoy nodes

### Docs

- [x] I've submitted a PR for a documentation page for the new node(s) that contains usage examples (see docs.flojoy.io)

### CODE REVIEW

File: VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
+from flojoy import flojoy, DataContainer
+import plotly.graph_objects as go
+import numpy as np
+import pandas as pd
+
+CELL_SIZE = 50
+FONT_SIZE = 10
+MAX_ALLOWED_SHAPE = 10
+MIN_ALLOWED_SHAPE = 4
+v_dot = "$\\vdots$"
```
Comment (JeffDotPng) [inline VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py:10]: 2 unused variables

File: VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        new_arr[: arr.size] = arr[: arr.size]
+    else:
+        new_arr = arr
+    return new_arr.reshape(-1, 1)
+
+
+@flojoy
+def ARRAY_VIEW(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+    """
+    It takes "ordered_pair", "dataframe", "matrix", and "image" as input type
```
Comment (JeffDotPng) [inline VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py:31]: The style guide states that every node docstring should start with "The {NAME} node..." (https://github.com/flojoy-io/nodes/blob/main/node_styleguide.md)

File: VISUALIZERS/DATA_STRUCTURE/MATRIX_VIEW/MATRIX_VIEW.py [hunk 3]
```diff
@@ -84,6 +84,20 @@ def numpy_array_as_table(arr: np.ndarray):
 
 @flojoy
 def MATRIX_VIEW(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+    """
+    It takes "matrix" or "ordered_pair" type as input type and displays its visualization
```
Comment (JeffDotPng) [inline VISUALIZERS/DATA_STRUCTURE/MATRIX_VIEW/MATRIX_VIEW.py:88]: Docstring thing again

File: VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+            data = dc_input.m
+            cell_values = numpy_array_as_table(data, l_dot)
+        case "image":
+            red = dc_input.r
+            green = dc_input.g
+            blue = dc_input.b
+
+            if dc_input.a == None:
+                merge = np.stack((red, green, blue), axis=2)
+                merge = merge.reshape(-1, merge.shape[-1])
```
Comment (JeffDotPng) [inline VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py:63]: Repeating 2 lines in both parts of the if statements can be moved out

File: VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+d_dot = "$\\ddots$"
+l_dot = "$\\ldots$"
+
+
+def numpy_array_as_table(arr: np.ndarray, placeholder: str):
+    if arr.size > MAX_ALLOWED_SHAPE:
+        converted_type = arr.astype(object)
+        new_arr = converted_type[:MAX_ALLOWED_SHAPE]
+        new_arr[MAX_ALLOWED_SHAPE - 2] = l_dot
+    elif arr.size < MIN_ALLOWED_SHAPE:
```
Comment (JeffDotPng) [inline VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py:20]: Is there a point to having a min allowed shape? Why not just display the whole array?

File: VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py (hunk 4 above)
Comment (Jin1L) [reply VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py:63]: @JeffDotPng, One of them, includes alpha, and the other doesn't. Do you still think I should move them out?

Comment (JeffDotPng) [reply VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py:57]: I'm talking about the lines `merge = merge.reshape(-1, merge.shape[-1])` and `cell_values = numpy_array_as_table(merge, l_dot)`

**Pull request merged**